xla/stream_executor/gpu/BUILD

```diff
@@ -904,3 +904,22 @@ xla_test(
         "@com_google_googletest//:gtest_main",
     ],
 )
+
+xla_test(
+    name = "gpu_kernel_args_test",
+    srcs = ["gpu_kernel_args_test.cc"],
+    backends = ["gpu"],
+    local_defines = if_cuda_is_configured(["GOOGLE_CUDA=1"]),
```

Comment:
Is this really needed? Can you maybe figure out at runtime whether it runs with cuda?